student_setup.py

```diff
@@ -0,0 +1,16 @@
+import sqlite3
+conn= sqlite3.connect("my-app.db")
+
+cursor = conn.cursor()
+
+student_data = [ ("Leonardo", 16), ("Donatello", 16), ("Raphael", 16), ("Michaelangelo", 16)]
+
+for t in student_data:
+	format_str = """INSERT INTO students (name, age) VALUES ("{name}", "{age}");"""
```

Comment:
This should be solved more generically. For the sake of simplicity consider only inserting one student at the moment. You have to be able to handle all possible inputs for students (they should match the columns on the student table in the database). Optional fields in the database should also be treated as optional in your code.